rpc/src/module/debug.rs

```diff
@@ -5,19 +5,19 @@ use std::time;
 #[rpc(server)]
 pub trait DebugRpc {
     #[rpc(name = "jemalloc_profiling_dump")]
-    fn jemalloc_profiling_dump(&self) -> Result<()>;
+    fn jemalloc_profiling_dump(&self) -> Result<String>;
```

Comment:
When `target_env = "msvc"`, `target_os="macos"` or feature `profiling` wasn't enable, it doesn't make sense to return a `String`.
https://github.com/nervosnetwork/ckb/blob/3f27eae7998ee24b282da54eb8d9cbe5b5e2d4cc/util/memory-tracker/src/lib.rs#L6-L10
https://github.com/nervosnetwork/ckb/blob/3f27eae7998ee24b282da54eb8d9cbe5b5e2d4cc/util/memory-tracker/src/jemalloc-mock.rs#L3-L5

Reply:
You are right, it is useless for these cases. I think it will be better to return an error message when the feature `profiling` is disabled. And it already updated, please take a look. Thanks.

Examples of the proposed responses of RPC jemalloc_profiling_dump:

  * Success : `{"jsonrpc":"2.0","result":"ckb-jeprof.1587964630.heap","id":2}`
  * Fail: `{"jsonrpc":"2.0","error":{"code":-32603,"message":"jemalloc profiling dump: unsupported"},"id":2}`

Reply:
I agree with u.
Please change the mock version of `jemalloc_profiling_dump`, let it return an error:
https://github.com/nervosnetwork/ckb/blob/e79f7ca87fbfbdd67e121c740dd7eafa9cc12f0a/util/memory-tracker/src/lib.rs#L15-L17